python/mxnet/gluon/block.py

```diff
@@ -1088,6 +1088,7 @@ def _get_graph(self, *args):
                 out = super().__call__(*args)
             flatten_out, self._out_format = _flatten(out, "output")
             symbol_outputs = dc.get_symbol(flatten_out, sym_cls=type(symbol_inputs[0]))
+            dc.clear(flatten_out)
```

Comment:
The `dc.clear` here is called on the ndarray outputs, which contain the node_entry(`deferredcompute_entry_`) with DCInfo. The copied symbol doesn't contain DCInfo, so I try to clear DCInfo from ndarray outputs.

Reply:
Sorry I misread the flatten_out as symbol_outputs. "so I try to clear DCInfo from ndarray outputs" still shouldn't be needed and garbage collection of the ndarray should clear the entry. So it would still be helpful to point out the temporary nature of the clear call